bundles/mapping/mapmodule/AbstractMapModule.js

```diff
@@ -2280,7 +2280,7 @@ Oskari.clazz.define(
                 layerFunctions = [],
                 sandbox = this.getSandbox();
 
-            if (!layer.isSupported(sandbox.getMap().getSrsName())) {
+            if (!layer.isSupported(sandbox.getMap().getSrsName()) && sandbox.mapMode !== 'mapPublishMode') {
```

Comment:
I'd really like to get rid of sandbox.mapMode as it's just some random flag the publisher uses without any proper documentation or anything. What happens without this?